### ISSUE TITLE
 No bookmarks found for the provided GUID. 

### ISSUE THREAD

**Issue by PeterPaul1234** [marcelklehr/floccus#690]
Version 4.4.0
Firefox 81.0.2 (64-Bit)
Win 10

**Comment by Stokel** [comment]
I'm getting the same error after the update when trying to sync to my nextcloud server with existing bookmarks from the earlier version.
It worked before the update.

- Floccus 4.4.0.1
- Archlinux
- Firefox 81.0.2
- Nextctloud Bookmarksapp 3.4.4
- Nextcloud 20.0.0

These are the last lines in my logfile:
```
2020-10-20T00:06:15.275Z (nextcloud-folders)REMOVE { bookmark:   - #undefined[Recent Tags](place:type=6&sort=14&maxResults=10) parentId: 5 }
2020-10-20T00:06:15.275Z (nextcloud-folders)REMOVE { bookmark:   - #undefined[](data:) parentId: 5 }
2020-10-20T00:06:15.275Z (nextcloud-folders)REMOVE { bookmark:   - #undefined[Most Visited](place:sort=8&maxResults=10) parentId: 6 }
2020-10-20T00:06:15.276Z (local)REMOVE     - #NKj5HRxZVat3[Recent Tags](place:type=6&sort=14&maxResults=10) parentId: menu________
2020-10-20T00:06:15.276Z (local)REMOVE     - #41seZf08_Zf8[](data:) parentId: menu________
2020-10-20T00:06:15.277Z (local)REMOVE     - #jz9OHHT1vtzZ[Most Visited](place:sort=8&maxResults=10) parentId: toolbar_____
2020-10-20T00:06:15.603Z [Error: No bookmarks found for the provided GUID.]
2020-10-20T00:06:15.603Z [Error: No bookmarks found for the provided GUID.]
2020-10-20T00:06:15.603Z [Error: No bookmarks found for the provided GUID.]
2020-10-20T00:06:15.603Z Syncing failed with No bookmarks found for the provided GUID.
No bookmarks found for the provided GUID.
No bookmarks found for the provided GUID.
2020-10-20T00:06:16.741Z onSyncFail
```

**Comment by marcelklehr** [comment]
I'm sorry for the inconvenience, this bug should be fixed in v4.4.1 -- you may need to start a sync from scratch in the account settings.

**Comment by vatolin** [comment]
The failure persists here as well with Floccus 4.4.6 at FF 82.03 at macOS 10.15.7 and Bookmarks 3.4.9 at NC 19.0.4.
